docs/src/hal/halshow.adoc

```diff
@@ -48,9 +44,7 @@ Notes:
        LinuxCNC must be running for standalone usage.
 ----
 
-Example to limit number of decimal points for floats
-and use a file named my.halshow in the current directory:
-
+.Example to limit number of decimal points for floats and use a file named my.halshow in the current directory:
```

Comment:
Having this all as a caption makes the code snippet a bit isolated (not referenced in text).

Reply:
changed back

Reply:
Reversed.